playbook/app/pb_kits/playbook/pb_button/_button_mixins.scss

```diff
@@ -68,7 +73,7 @@ $pb_button_border_width: 0px;
 
 // Primary ======================
 @mixin pb_button_primary {
-  @include pb_button;
+  @include variant_overrides;
```

Comment:
It's kind of strange that `variant_overrides` doesn't pass any overrides. Thoughts? I feel a little less confident in my understanding when I see these. Should we change the name?

Reply:
Yeah it's odd, the mixin itself has default arguments.  Unfortunately need them for some mixin implementations where arguments are specified.

I will see if I can just pass them explicitly on line 76 above though so it's more intuitive.